book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/2_0_3_docker_and_portainer.md

```diff
@@ -35,11 +35,11 @@ Until everything is downloaded the Portainer interface will not start.
 
 To debug this, login to the robot:
 
-    laptop $ ssh ![hostname]
+    laptop $ ssh ![linux-username]@![hostname].local
```

Comment:
From the previous steps `ssh hostname` should work.

Reply:
Doesn't work for me but I might have messed up my configuration at some point.